.github/workflows/build_icons.yml

```
@@ -19,10 +19,7 @@ jobs:
           pip install -r ./.github/scripts/requirements.txt
           npm install
       - name: Run icomoon_upload.py
```

Since we are not having the `icomoon_upload.py` script anymore, this step should be renamend imo:
```suggestion
      - name: Executing build and create fonts via icomoon
```